packages/react/components/table/src/TableColumn/TableColumn.styles.ts

```diff
@@ -22,6 +22,14 @@ export const StyledTableColumn = styled('th', {
     bottom: 'calc(50% - 14px)',
   },
 
+  '&:first-child': {
+    borderRight: '2px solid $colors$border-primary',
+  },
+
+  '&:last-child': {
+    borderLeft: '2px solid $colors$border-primary',
+  },
```

Comment:
are these always going to have borders, or is that for sticky columns when we introduce it?

Reply:
I'll have to let @andrea-p44 weigh in on that

Reply:
I confirmed with @andrea-p44, this will come in our datatable implementation. Our base table wont have the concept of sticky

Reply:
If I understand, shall I remove this border entirely
<img width="911" alt="Screenshot 2023-03-06 at 12 16 11 PM" src="https://user-images.githubusercontent.com/3459902/223196206-aad20262-361b-49ab-b883-791010d0dc47.png">
?